src/main/java/ca/uhn/fhir/jpa/starter/model/ApiAsyncTaskEntity.java

```diff
@@ -35,13 +35,17 @@ public enum Status {
 	@Column(name = "lastUpdated", nullable = true)
 	private Date lastUpdated;
 
+	@Column(name = "isAnonymousEntry", nullable = true)
+	private String isAnonymousEntry;
+
 	public ApiAsyncTaskEntity() {}
-	public ApiAsyncTaskEntity(String uuid, String status, java.sql.Clob  summaryResult,java.sql.Clob dailyResult, Date lastUpdated) {
+	public ApiAsyncTaskEntity(String uuid, String status, java.sql.Clob  summaryResult,java.sql.Clob dailyResult, Date lastUpdated, String isAnonymousEntry) {
```

Comment:
```suggestion
	public ApiAsyncTaskEntity(String uuid, String status, java.sql.Clob  summaryResult,java.sql.Clob dailyResult, Date lastUpdated, boolean isAnonymousEntry) {
```
isAnonymousEntry can be of type boolean

Reply:
Making it boolean will store the value as hex in db 0x00 for false and 0x01 for true. Instead of doing I have created an enum for this.